browser-test/src/admin/admin_program_migration.test.ts

```diff
@@ -37,14 +81,18 @@ test.describe('program migration', {tag: ['@uses-fixtures']}, () => {
     })
 
     await test.step('import a program', async () => {
-      // TODO(#7087): We should also have a test that exports JSON and then imports that same JSON
-      // so that we can verify export and import work together.
       const sampleJson = readFileSync(
         'src/assets/import-program-sample.json',
         'utf8',
       )
       await adminProgramMigration.submitProgramJson(sampleJson)
 
+      await adminProgramMigration.expectProgramImported('Import Sample Program')
+      // The import page currently shows question IDs, so this screenshot needs
```

Comment:
Does https://github.com/civiform/civiform/wiki/Testing#de-randomizing help with this?